### PR TITLE
Add doctor to integration tests.

### PR DESCRIPTION
**Pull request by wgrzelak**

### CODE REVIEW

File: tests/marketplace/deployer_helm_tiller_base/onbuild/standard/run_test [hunk 1]
```diff
@@ -22,6 +22,9 @@ docker run \
   > dev
 chmod a+x dev
 
+# Runs diagnostic tool to make sure your environment is properly setup.
```
Comment (huyhg) [inline tests/marketplace/deployer_helm_tiller_base/onbuild/standard/run_test]: Can you do this for the other tests as well please? All of them are under `/tests/marketplace/`

Comment (wgrzelak) [reply tests/marketplace/deployer_helm_tiller_base/onbuild/standard/run_test]: Done.

**Comment by wgrzelak** [comment]
@huyhg can you review?

File: tests/marketplace/deployer_envsubst_base/full/app.Makefile [hunk 2]
```diff
@@ -132,7 +132,7 @@ app/verify_istio: app/build \
 # Runs diagnostic tool to make sure your environment is properly setup.
 app/doctor: | .build/app/dev
 	$(call print_target)
-	.build/app/dev /scripts/doctor.py
+	.build/app/dev /scripts/doctor
```
Comment (huyhg) [inline tests/marketplace/deployer_envsubst_base/full/app.Makefile]: drop `scripts/`

**Pull request merged**